RobotCode/src/main/java/frc/robot/Subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,81 @@
+package frc.robot.Subsystems;
+
+// import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.revrobotics.CANSparkBase.IdleMode;
+
+import frc.robot.constants.Constants;
+import frc.robot.constants.Ports;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.lib.core.motors.TeamSparkMAX;
+
+public class ShooterSubsystem extends SubsystemBase
+{
+	// creates objects
+	private double shooterMotorPower;
+	private double feederMotorPower;
+
+	private PIDController shooterPID;
```

Comment:
Not sure we need PID on these motors, but it's fine for now. Condense into a single line

Reply:
Also make final